eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/service/connector/ConnectorDataService.java

```diff
@@ -17,9 +17,14 @@
 
 package org.apache.eventmesh.dashboard.console.service.connector;
 
+import org.apache.eventmesh.dashboard.console.entity.connector.ConnectorEntity;
+
+import java.util.List;
+
 /**
  * Service providing data of connectors.
  */
```

Comment:
Could you please specify what type of data is processed in this service?

Reply:
Because `data` is a quite broad concept, it'll be better to be precise about the scope of the concept in the naming.